packages/coding-standard/docs/phpstan_rules.md

```diff
@@ -1659,6 +1659,61 @@ final class ProductRepository
 
 <br>
 
+## Forbid Protected Property Usage. Use Interface Instead.
+
+- class: [`ForbiddenProtectedPropertyRule`](../src/Rules/ForbiddenProtectedPropertyRule.php)
+
+```yaml
+# phpstan.neon
+services:
+    -
+        class: Symplify\CodingStandard\Rules\ForbiddenProtectedPropertyRule
+        tags: [phpstan.rules.rule]
+```
+
+```php
+<?php
+
+
+declare(strict_types=1);
+
+abstract class SomeClass
+{
+    protected $config = [];
+}
+```
+
+:x:
+
+```php
+<?php
+
+
+declare(strict_types=1);
+
+interface ConfigInterface
+{
+    public function config():
+}
+
+abstract class SomeClass implements ConfigInterface
+{
+    public function config()
+    {
+        // ...
+    }
+
+    public function run()
+    {
+        $this->config();
```

Comment:
Better add prefix to show there is return value

```suggestion
        $this->getConfig();
```

Reply:
updated.